src/graphql/query/featuredLoansData.graphql

```diff
@@ -25,5 +26,21 @@ query featuredLoansData(
 				}
 			}
 		}
+		loans(
```

Comment:
Would it be much effort to do this as a separate query and select which loan query to call based on the experiment assignment?

Reply:
I think longer term I'd like to have a data provider we could alter params on to switch queries. But for this one, it was just more simple to run both to get to ensure this loan was rendered from the server using the prefetch.